<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionAvion</class>
 <widget class="QMainWindow" name="gestionAvion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SKYUP AIRLINE</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 85, 127);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; color:#ffffff;&quot;&gt;Gestion des Avions&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="gestionavion_tab">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>701</width>
      <height>431</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="AjouterAvion">
     <attribute name="title">
      <string>Ajouter un avion</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>70</y>
        <width>103</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant avion :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>120</y>
        <width>103</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Référence avion :Référence avion :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>160</y>
        <width>103</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom avion :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>210</y>
        <width>103</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Date d'entrée :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>250</y>
        <width>103</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Etat de l'avion  :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>300</y>
        <width>150</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre de places avion  :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idavion">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>63</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="refavion">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>110</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nomavion">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="etatavion">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>240</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="nbplaceavion">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>293</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateentreeavion">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>200</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouteravion">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>370</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>269</x>
        <y>9</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#5555ff;&quot;&gt;Ajouter un avion &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="textSetting">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>340</y>
        <width>621</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Supprimeruneavion">
     <attribute name="title">
      <string>Supprimer un avion</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>269</x>
        <y>29</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; color:#5555ff;&quot;&gt;Supprimer un avion &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idavion_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>83</y>
        <width>471</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>103</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Identifiant avion:&lt;/p&gt;&lt;p/&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_gestionAvion_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>631</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="supprimerAvion">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>370</y>
        <width>88</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QLabel" name="testSupp">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>621</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Consulterlesavions">
     <attribute name="title">
      <string>Consulter les avions</string>
     </attribute>
     <widget class="QTableView" name="tableView_gestionAvion">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>611</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Retour</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
